var/spack/repos/builtin/packages/gplates/package.py

```diff
@@ -34,13 +36,9 @@ class Gplates(CMakePackage):
     depends_on('boost+python@1.34:1.60')
     depends_on('python@2:2.99')
 
-    # Officially, GPlates only supports GCC 4. Moreover, it requires
-    # QtXmlPatterns, which Qt 4 only builds with GCC 4.
-    conflicts('%gcc@5:')
```

Comment:
why did you remove this? Is the comment wrong? Please add that in the commit message